controller-server/src/main/java/com/yahoo/vespa/hosted/controller/athenz/mock/AthenzClientFactoryMock.java

```diff
@@ -33,6 +35,11 @@ public AthenzDbMock getSetup() {
         return athenz;
     }
 
+    @Override
```

Comment:
Why does this class extend AbstractComponent?

Reply:
I do not recall

Reply:
I think you should remove it then..